lib/internal/process/per_thread.js

```diff
@@ -258,6 +259,21 @@ function wrapProcessMethods(binding) {
     };
   }
 
+  /**
+   * Loads the `.env` file to process.env.
+   * @param {string | URL | Buffer | undefined} path
+   */
+  function loadEnvFile(path = undefined) {
+    if (path != null) {
+      const { getValidatedPath } = require('internal/fs/utils');
+      const { toNamespacedPath } = require('path');
```

Comment:
This should be moved to the top of the file, otherwise, monkey-patching will expose a vulnerability.

Reply:
Nice catch. Thanks!